int8 bit
bool cmd

---

bool success
